test/systemtests/docker_test.go

```diff
@@ -67,6 +67,12 @@ func (d *docker) runContainer(spec containerSpec) (*container, error) {
 		netstr = "--net=" + netstr
 	}
 
+        // Handle the case of "default-group" not specified explicitly 
+        // but only for the aci case (detected here via special aci network name)
+        if (spec.serviceName == "") && (spec.networkName == "default-net-aci"){
+                netstr = "--net=default-group"
```

Comment:
Why do we have this? This defeats the purpose of the test. The whole point is that in ACI mode, the code you added takes care of this automatically.